webpack.dev.js

```diff
@@ -39,7 +39,13 @@ module.exports = {
 			'commonjs2': 'react-dom/server',
 			'amd': 'react-dom/server',
 			'root': 'ReactDOMServer'
-		}
+		},
```

Comment:
Let's just stick with whatever is there for now, I'll run the linter over everything later. Same with the whitespace changes in the component.